public/app/plugins/datasource/graphite/configuration/MappingsConfiguration.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ChangeEvent, useState } from 'react';
+import { Button, Icon, InlineField, InlineFieldRow, Input } from '@grafana/ui';
+import MappingsHelp from './MappingsHelp';
+
+type Props = {
+  mappings: string[];
+  onChange: (mappings: string[]) => void;
+  onDismiss: () => void;
+  onRestoreHelp: () => void;
+  showHelp: boolean;
+};
+
+export const MappingsConfiguration = (props: Props): JSX.Element => {
+  const [mappings, setMappings] = useState(props.mappings || []);
+
+  return (
+    <div>
+      <h3 className="page-heading">Label mappings</h3>
+      {!props.showHelp && (
+        <p>
+          <a className="external-link" target="_blank" rel="noreferrer" onClick={props.onRestoreHelp}>
```

Comment:
Is there a reason why we are not using `<Button variant="link />` ?

Reply:
The only reason is that I didn't know we had it. Thanks! I've changed it to a proper link button :)